packages/jest-cli/src/jest.js

```diff
@@ -282,6 +287,7 @@ const runCLI = (
                   ? chalk.dim(' \u203A Press ') + 'u' + chalk.dim(' to update failing snapshots.')
                   : null,
                 chalk.dim(' \u203A Press ') + 'p' + chalk.dim(' to filter by a filename regex pattern.'),
+                chalk.dim(' \u203A Press ') + 'i' + chalk.dim(' to interrupt watch mode.'),
```

Comment:
Sure, `i` works better for something like `interactive` I now think, but I had this `interruption` in mind. Updated to `c` and cancel as requested